src/net.cpp

```diff
@@ -144,7 +144,7 @@ static std::vector<CAddress> convertSeed6(const std::vector<SeedSpec6> &vSeedsIn
     {
         struct in6_addr ip;
         memcpy(&ip, i->addr, sizeof(ip));
-        CAddress addr(CService(ip, i->port));
+        CAddress addr(CService(ip, i->port), NODE_NETWORK);
```

Comment:
@AloeareV I am still skeptical about the definition of `NODE_NETWORK`:

`NODE_NETWORK = (1 << 0)`

but your hypothesis that it's to make it explicit that the value is to be used as a bit-flag is consistent with other patterns we've seen in C++ code (e.g. type hints in names), so it's plausible.

Reply:
https://www.stroustrup.com/C++11FAQ.html <-- Let's stay aware of the `strong enum` pattern.